app/bundles/DynamicContentBundle/EventListener/LeadSubscriber.php

```diff
@@ -54,8 +54,8 @@ public static function getSubscribedEvents()
     public function onTimelineGenerate(LeadTimelineEvent $event)
     {
         // Set available event types
-        $eventTypeKey      = 'dynamic.content.sent';
-        $eventTypeNameSent = $this->translator->trans('mautic.dynamic.content.sent');
+        $eventTypeKey      = 'dynamic.content.triggered';
```

Comment:
I assume this is BC, key should stay same like before. Probably somebody use it in plugin and it could break it

Reply:
Thank you! I have restore the key. 
I thought this key was only used in the code to generate and display the timeline and could not be the cause of BC. Maybe I have missed smth.